### ISSUE TITLE
Some essay questions

### ISSUE THREAD

**Issue by zuiluorenjian** [skJack/DiffusionFake#8]
The SBI data in the comparative experiment in your paper is completely different from the SBI experimental data in the original paper. Did you use any different data processing methods?

**Comment by skJack** [comment]
Thank you for your attention. To ensure the fairness of the test, I used the official code and model 
for testing. The data and other conditions of the diffusionfake method are exactly the same. (The inference method in the SBI paper is completely different from ours)

**Comment by zuiluorenjian** [comment]
I don't quite understand. What I mean is that in the 2022CVPR (SBI) paper experiment, Celeb-DF performed very well, but in your paper experiment table, SBI only had 80% AUC on Celeb-DF. Other datasets are similar. Did you use the c40 version of FF++ as training data to cause such a result? I'm a novice and may not know much.

**Comment by skJack** [comment]
No, I used the official c23 model for testing. I think the main performance difference comes from the test method. SBI uses the average of each video frame (kindly refer to the SBI paper: Inference Strategy. We sample 32 frames per video for
inference. If two or more faces are detected in a frame, the classifier is applied to all faces and the highest fakeness confidence is used as the predicted confidence for the frame. Once the predictions for all frames are obtained, we average them to get the prediction for the video. For a fair comparison, we use all videos of all test sets for evaluation by setting the confidences to 0.5 for the videos where no face is detected in all frames.) 
This average will make the performance higher, but our test method is to directly calculate the AUC of each image, which will cause performance differences, so we use the official checkpoint to re-test on our test set to ensure fairness in the comparison.

**Comment by zuiluorenjian** [comment]
OK, I understand, thanks for the reply.
